rotkehlchen/api/v1/schemas.py

```diff
@@ -492,35 +519,37 @@ def validate_asset_movements_query_schema(  # pylint: disable=no-self-use
             data: Dict[str, Any],
             **_kwargs: Any,
     ) -> None:
-        value = data['order_by_attribute']
-        if data['order_by_attribute'] not in (
-                None,
-                'timestamp',
-                'location',
-                'category',
-                'amount',
-                'fee',
-        ):
-            raise ValidationError(
-                message=f'order_by_attribute for asset movements can not be {value}',
-                field_name='order_by_attribute',
-            )
+        if data['order_by_attribute'] is None:
+            return
+        valid_ordering_attr = {
+            None,
+            'timestamp',
+            'location',
+            'category',
+            'amount',
+            'fee',
+        }
+        for attr in data['order_by_attribute']:
```

Comment:
same thing as above for subset. Also question if you use just tuple in and not set operations what is the point of turning a tuple into a set?